packages/shared/src/utils/constants/mt5-login-list-status.ts

```diff
@@ -0,0 +1,6 @@
+export const MT5AccountStatus = Object.freeze({
```

Comment:
**NIT**: TitleCase is only used for class or components. For constants we use `snake_case` with CAPS